sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -17,6 +17,7 @@ public abstract partial class AsyncPageable<T> : System.Collections.Generic.IAsy
     }
     public static partial class AzureCoreExtensions
     {
+        public static System.Collections.Generic.IDictionary<string, object?> ToDictionary(this System.BinaryData data) { throw null; }
```

Comment:
I think we should call it ToDictionaryFromJson. There is a method ToObjectFromJson on BinaryData and this should use the same naming.